polkadot-protocol/basics/chain-data.md

```diff
@@ -0,0 +1,183 @@
+-d '{"id":1, "jsonrpc":"2.0", "method": "state_getMetadata"}' \
+https://rpc.polkadot.io
+
+```
+
+### Use Subxt
+
+[`subxt`](https://github.com/paritytech/subxt){target=\_blank} may also be used to fetch the metadata of any data in a human-readable JSON format: 
+
+```sh
+
+subxt metadata  --url wss://rpc.polkadot.io --format json > spec.json
+
+```
+
+Another option is to use the [`subxt` explorer web UI](https://paritytech.github.io/subxt-explorer/#/){target=\_blank}.
+
+## Client Applications and Metadata
+
+The metadata exposes the expected way to decode each type, meaning applications can send, retrieve, and process application information without manual encoding and decoding. Client applications must use the [SCALE codec library](https://github.com/paritytech/parity-scale-codec?tab=readme-ov-file#parity-scale-codec){target=\_blank} to encode and decode RPC payloads to use the metadata. Client applications use the metadata to interact with the node, parse responses, and format message payloads sent to the node.
+
+## Metadata Format
+
+Although the SCALE-encoded bytes can be decoded using the `frame-metadata` and [`parity-scale-codec`](https://github.com/paritytech/parity-scale-codec){target=\_blank} libraries, there are other tools, such as `subxt` and the Polkadot-JS API, that can convert the raw data to human-readable JSON format.
+
+The types and type definitions included in the metadata returned by the `state_getMetadata` RPC call depend on the runtime's metadata version.
+
+In general, the metadata includes the following information:
+
+- A constant identifying the file as containing metadata
+- The version of the metadata format used in the runtime
+- Type definitions for all types used in the runtime and generated by the `scale-info` crate
+- Pallet information for the pallets included in the runtime in the order that they are defined in the `construct_runtime` macro
+
+!!!warning "Metadata formats may vary"
+    Depending on the frontend library used (such as the [Polkadot API](https://papi.how/){target=\_blank}), they may format the metadata differently than the raw format shown.
+
+The following example illustrates a condensed and annotated section of metadata decoded and converted to JSON:
+
+```json
+
+--8<-- 'code/polkadot-protocol/basics/chain-data/metadata-format.json'
+
+```
+
+The constant `1635018093` is a magic number that identifies the file as a metadata file. The rest of the metadata is divided into the `types`, `pallets`, and `extrinsic` sections:
+
+- The `types` section contains an index of the types and information about each type's type signature
+- The `pallets` section contains information about each pallet in the runtime
+- The `extrinsic` section describes the type identifier and transaction format version that the runtime uses
+
+Different extrinsic versions can have varying formats, especially when considering [signed transactions](TODO: update path). 
+
+### Pallets
```

Comment:
I feel like we could do a little more to explain metadata a little more granularly component by component, and even show where you would use each field - but this can happen in the future in another page ("Using Metadata", or something)